libs/shared/src/consts/providers/provider.enum.ts

```diff
@@ -27,7 +27,7 @@ export enum EmailProviderIdEnum {
   Mailgun = 'mailgun',
   Mailjet = 'mailjet',
   Mandrill = 'mandrill',
-  Nodemailer = 'nodemailer',
+  CustomSMTP = 'custom_smtp',
```

Comment:
I am afraid this could be a breaking change as in the database as providerId our users would have stored `nodemailer`. What we can do for now is to just change the key but not the value and update the rest of the codebase accordingly.
```suggestion
  CustomSmtp = 'nodemailer',
```
@davidsoderberg @scopsy what do you think? I am having a wrong consideration?

Reply:
Good catch